lib/services/peters.js

```diff
@@ -7,47 +7,73 @@ var moment = require('moment')
 module.exports = function () {
   var x = xray()
 
-  var processMenu = function (next, date) {
+  var map = {
+    leden: '01',
+    únor: '02',
+    březen: '03',
+    duben: '04',
+    květen: '05',
+    červen: '06',
+    červenec: '07',
+    srpen: '08',
+    září: '09',
+    říjen: '10',
+    listopad: '11',
+    prosinec: '12'
+  }
+
+  var parseDate = function (dateString) {
+    return moment(dateString.replace(/^.+,\s+/, '').replace(' (dnes)', '').replace(new RegExp(_.keys(map).join('|')),
+      function (replacer) {
+        return map[replacer]
+      }), 'D M')
```

Comment:
funkce fajn, ale trochu bych ji zčitelnil

- každý replace na vlastní řádek
- replacer vytáhnout s mapou mimo module.exports, to se nemusí vytvářet pokaždý znova